docs/Developing-Wallace-(setup-guide).md

```diff
@@ -10,7 +10,7 @@ python --version
 ```
 If you do not have Python 2.7 installed, you can install it from the [Python website](https://www.python.org/downloads/).
 
-Or, if you use Homebrew:
+Or, if you use Homebrew (if you use Anaconda, install using `conda`, not Homebrew):
```

Comment:
> I'm pretty sure if you didn't have 2.7 already installed, it would then
install it, though I can't actually find the documentation describing that
and don't want to create a clean wipe of my system to test this.

Yes, that's correct.

On Wed, Aug 10, 2016 at 8:51 AM, Michael Pacer <notifications@github.com>
wrote:

> In docs/Developing-Wallace-(setup-guide).md
> <https://github.com/berkeley-cocosci/Wallace/pull/430#discussion_r74199802>
> :
>
> > @@ -10,7 +10,7 @@ python --version
> >  ```
> >  If you do not have Python 2.7 installed, you can install it from the [Python website](https://www.python.org/downloads/).
> >
> > -Or, if you use Homebrew:
> > +Or, if you use Homebrew (if you use Anaconda, install using `conda`, not Homebrew):
>
> Yes if you need a different version:
>
> paraphrasing http://conda.pydata.org/docs/py2or3.html#update-or-upgrade-
> python
>
> conda install python=3.5 will upgrade your python in whatever branch you
> are in to python3.5 (and will still allow you to create new environments
> with your previous python version)
>
> but more generally if you were to create a new conda environment with
> Python 2.7 using
>
> conda create -n py27 python=2.7
>
> I'm pretty sure if you didn't have 2.7 already installed, it would then
> install it, though I can't actually find the documentation describing that
> and don't want to create a clean wipe of my system to test this.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/berkeley-cocosci/Wallace/pull/430/files/7d36f8ef2b441d2eedb3bf4f91cecb5d87004143#r74199802>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFF9DTATPW8m-IGqn50e1h7XO1dalEDks5qeYL1gaJpZM4JfvhD>
> .
>